### ISSUE TITLE
Only receiving 3 frames that count up?

### ISSUE THREAD

**Issue by audioeptesicus** [uhi22/wifican#4]
Thanks for your efforts.  I'm using an Espressif ESP32-S3-DevKitC-1 board with a Waveshare SN65HVD230.  I have wired everything as mentioned, however, this is all I see in SavvyCAN.

![image](https://github.com/user-attachments/assets/d4f3e678-6f11-4820-9d1a-24bfd35ee1d0)

0x59E is read when I'm connected to the OBD2 port, but otherwise goes away.  Regardless of whether or not I'm connected to the OBD2 port, 1000-1002 continue to climb in value on each read frame.  Where are these coming from?  So far, I have not been able to receive any frames on the CAN bus from my vehicle, a 2023 Ford F350.

**Comment by uhi22** [comment]
Good point, I need to extend the readme. The IDs 1000 to 1002 are just debug data, you can load the dbc into Savvycan to decode the content.
If you see the 59E the connection when connecting to the vehicle is fine. Normally the OBD CAN is silent, as long as no diagnostic tool is connected. So you see more then I would have expected. You could  run TorquePro or carscanner app with a Bluetooth OBD connector, then the Savvycan should show the diagnostic requests and responses.
If other CANs of the car do not show data, they may have different baud rate. Never tested whether changing the baudrate via Savvycan works.

**Comment by audioeptesicus** [comment]
Thanks so much for the response and clarification!  I know this goes outside of the 'issue', but if I'm able to find the CAN data for the 4 different door statuses using the wifican with another OBD2 app like Torque or Forscan, then how will I be able to read this data in the future?  I'm attempting to build a little ESP32 controller to read the open/close status of my doors for some power running boards.

**Comment by uhi22** [comment]
The OBD apps will NOT help to find these signals on the plain CAN. Because OBD uses high level protocols like ISOTP and UDS. This is in contrast to the pure CAN messages, where a certain signal (e.g. driver door open) is mapped to a certain position of a certain message.
The usual way is to NOT use the OBD CAN (because it is normally quite silent). Search for the body CAN, eg at the door connector. If Savvycan shows lots of different messages then you are right. Stare at the messages while you are opening and closing a door, there must be a related bit in a certain message.

If this was successful, you could (as quick and dirty solution) add some code to the wifican sources to evaluate certain bits of certain messages, and e.g. control a digital output.

Another way to find out the signals of interest is googling "dbc" for your type of car. Maybe someone published the data description already.
